o-markitdigital-equity-highlight/main.scss

```diff
@@ -1,12 +1,20 @@
 // Output grid helper classes and data-attributes
+
+$o-typography-is-silent: false;
+$o-fonts-is-silent: false;
+$o-colors-is-silent: false;
```

Comment:
Setting o-colors-is-silent to false here will increase the size of your bundle a lot because o-colors will put out heaps and heaps of concrete classes which I doubt you're using ie: https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-colors@^4.0.0